apps/account-worker/src/transaction_notifier/notifier.service.ts

```diff
@@ -121,7 +121,6 @@ export class TxnNotifierService
         .filter(({ event }) => this.blockchainService.events.capacity.CapacityWithdrawn.is(event))
         .reduce((sum, { event }) => (event as unknown as any).data.amount.toBigInt() + sum, 0n);
 
-      // eslint-disable-next-line no-restricted-syntax
```

Comment:
I think the airbnb package must have defined some non-default restricted syntax. I turned it on with just the default settings.